adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adopt a Dog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  
+</head>
+<body>
+    <header>
+            <div class="title">
+                    <!-- <img src = "../adopt-a-dog/images/christopher-harris-57366-unsplash.jpg" alt="brown dog  face in the shadows"  alt = "Christoper harris"class = "header-logo">  -->
+                <img class = "logo" src="adoptadog-logo.svg" alt="ADOPTADOG" width = "400">
+                <h1>Seattle's premiere dog adoption agency</h1>
+            </div>
+                <!-- <div class="checkout">
+                    <h6>shopping cart $1234.56</h6>
+                    <h6>Checkout</h6>
+                </div> -->
+    </header>
+                <nav class = "main-nav">
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="dogs.html">Dogs</a></li>
+                        <li><a href="blog.html">Blog</a></li>
+                    </ul>
+                </nav>
+      <article>
+        <div class="grid-container">
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/alex-lacayo-756508-unsplash.jpg" alt="Murphy">
+              <h5>Murphy</h5>
+              <p>Cost to Adopt: $123.45 
+              </p>
+                <p>Corrum volorit iandae
+                  nimaxim cum restia volor
+                  reicid ut et etur sunt arum
+                  rendae pla endis re ea erum,
+                  qui doluptae
+                </p>
+            </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/nikolay-tchaouchev-785309-unsplash.jpg" alt="Poppy">
+              <h5>Poppy</h5>
+              <p>Cost to Adopt: $123.45
+              </p>
+                  <p>Corrum volorit iandae
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+            </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/marcus-wallis-471438-unsplash.jpg" alt="Jack">
+              <h5>Jack</h5>
+              <p>Cost to Adopt: $123.45
+              </p>
```

Comment:
make sure your p tags are either on the same line, or the text inside is indented in one level. Both are valid, but be consistent. I prefer the first option, but it's six of one, half a dozen of the other.

<p>words about stuff</p>

or

<p>
  words about stuff
</p>